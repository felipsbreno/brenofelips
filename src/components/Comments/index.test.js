import React from "react"
import { render, screen } from "@testing-library/react"
import Comments from "."

describe("Comments", () => {
  it("should render equals snapshot", () => {
    const { container } = render(<Comments />)
    expect(container).toMatchSnapshot()
  })

  it("should be component Comments", () => {
    render(<Comments />)

    const getElementById = screen.getByTestId("comments")
    expect(getElementById).toBeInTheDocument()
  })

  it("should be render text", () => {
    render(<Comments />)

    const text = screen.getByText("Coment√°rios")
    expect(text).toBeInTheDocument()
  })
})
