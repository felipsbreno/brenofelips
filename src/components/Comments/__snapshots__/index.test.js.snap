// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Comments should render equals snapshot 1`] = `
.c0 {
  margin: auto;
  max-width: 70rem;
  padding: 3rem 6.4rem 3rem;
}

.c0 iframe[src*="ads-iframe"] {
  display: none;
}

.c0 #disqus_thread a {
  color: var(--highlight)!important;
}

.c1 {
  color: var(--postColor);
  font-size: 2.1rem;
  font-weight: 700;
  padding-bottom: 2rem;
}

@media (max-width: 1170px) {
  .c0 {
    padding: 3rem 1.4rem 0;
    max-width: 100%;
  }
}

@media (max-width: 1170px) {
  .c1 {
    font-size: 1.375rem;
  }
}

<div>
  <section
    class="c0"
    data-testid="comments"
  >
    <h2
      class="c1"
    >
      Coment√°rios
    </h2>
    <div>
      <div
        id="disqus_thread"
      />
    </div>
  </section>
</div>
`;
